BMM.Api/Utils/SongbookUtils.cs

```diff
@@ -1,20 +1,14 @@
-using System.Collections.Generic;
-
 namespace BMM.Api.Utils
 {
     public static class SongbookUtils
     {
-        private static IDictionary<string, string> NameMappingDictionary { get; } = new Dictionary<string, string>()
-        {
-            { "herrens_veier", "HV" },
-            { "mandelblomsten", "FMB" },
-        };
-
         public static string GetShortName(string name)
         {
-            return NameMappingDictionary.TryGetValue(name, out string shortName)
-                ? shortName
-                : string.Empty;
+            if (name == "herrens_veier")
+                return "HV";
+            if (name == "mandelblomsten")
+                return "FMB";
+            return name;
```

Comment:
Isn't is the same as just 

return NameMappingDictionary.TryGetValue(name, out string shortName)
                ? shortName
                : name;

Reply:
yes. I just figured that above if structure is easier to read.